storage/buckets/buckets_test.go

```diff
@@ -31,42 +32,77 @@ import (
 	iampb "google.golang.org/genproto/googleapis/iam/v1"
 )
 
+const (
+	testPrefix      = "storage-buckets-test"
```

Comment:
Let's include the projectID in this as well if possible?

Reply:
We could include it here, but it looks like the individual tests should still call `testutil.SystemTest(t)` to skip them if the env variable for the project isn't set, and also gives us that project id. So adding a global projectID here wouldn't be very useful unless we want to refactor that part as well